### PR TITLE
Implement option to save last frame for OpenGL

### PR DESCRIPTION
**Pull request by ryanmccauley211**
## Overview: What does this pull request change?

Implemented save_last_frame option for opengl

## Motivation and Explanation: Why and how do your changes improve the library?

#1936

Implement option to save last frame as an image for opengl
## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by Darylgolden** [comment]
Works for me on Windows, not really sure why the tests are failing?

**Comment by ryanmccauley211** [comment]
> Works for me on Windows, not really sure why the tests are failing?

Yea I am not sure, but as discussed I removed the test for now

**Review by Darylgolden — APPROVED** [review]
LGTM, let's wait on opinions from more devs to see if they're ok with omitting the test.

File: manim/renderer/opengl_renderer.py [hunk 1]
```diff
@@ -416,8 +417,30 @@ def update_frame(self, scene):
...
+        """
+        image = Image.frombytes(
+            "RGBA",
+            self.get_pixel_shape(),
+            self.context.fbo.read(self.get_pixel_shape(), components=4),
+            "raw",
+            "RGBA",
+            0,
+            -1,
+        )
```
[the remark this answers is not in the document]
Comment (ryanmccauley211) [reply manim/renderer/opengl_renderer.py:446]: Added doc strings explaining it. RGBA is the mode of the image, 'raw' is the decoder to use and the second RGBA is the mode the decoder should use when reading the bytes. 0 means there is no padding between bytes and -1 represents orientation, which seems to need to be reversed, although I'm not sure why this is, but if you change it to 1 it looks like a mirror image.

**Pull request merged**